internal/models/network.go

```diff
@@ -100,7 +115,11 @@ type DeviceFingerprintCloudEvent struct {
 
 type DeviceDataStatusCloudEvent struct {
 	CloudEventHeaders
-	Data DeviceStatusData `json:"data"`
+	Data    DeviceStatusData `json:"data"`
+	TokenId int64            `json:"tokenId"`
```

Comment:
I added tokenId and Make, Model and Year only to the DeviceDataStausCloudEvent. Should we add it to the other events as well?

Reply:
this is good, for now only needed for Status. 
TokenId we'll see if we add to other payloads later.